.github/ISSUE_TEMPLATE/improve-the-site.yml

```diff
@@ -0,0 +1,62 @@
+name: Improve the docs.github.com site
```

Comment:
I often get tripped up by this name @janiceilene. Maybe I'm alone but something like "Share feature ideas and bugs" would seem more obvious to me.

Reply:
I agree that this isn't the clearest name! We mostly try to avoid the word "bug," but this one is supposed to be used for the site instead of content. Maybe "Suggest ideas and updates to the Docs website" or something?

Reply:
"Suggest ideas and updates for the Docs website" seems clear to me 👍

Reply:
```suggestion
name: Suggest ideas and updates for the Docs website
```